app/src/main/java/org/greenstand/android/TreeTracker/messages/individualmeassagelist/IndividualMessageListScreen.kt

```diff
@@ -125,5 +126,8 @@ fun IndividualMessageListScreen(
                 }
             }
         }
+        if (state.messages.isEmpty()) {
```

Comment:
This looks good overall. Let's make sure we don't run the list code when it's empty.

Reply:
> This looks good overall. Let's make sure we don't run the list code when it's empty.

Sorry I don't get this

Reply:
Let's not run the code to render a list of it's empty

Reply:
If list empty show empty state, else show list

Reply:
Oh I get you now.